syncd/SaiSwitch.cpp

```diff
@@ -744,6 +744,25 @@ std::set<sai_object_id_t> SaiSwitch::getWarmBootDiscoveredVids() const
     return m_warmBootDiscoveredVids;
 }
 
+void SaiSwitch::redisSaveInternalOids(_In_ sai_object_id_t rid) const
+{
+    SWSS_LOG_ENTER();
+
+    std::set<sai_object_id_t> coldVids;
+
+    sai_object_id_t vid = m_translator->translateRidToVid(rid, m_switch_vid);
+
+    coldVids.insert(vid);
+
+    m_client->setDummyAsicStateObject(vid);
+
+    m_client->saveColdBootDiscoveredVids(m_switch_vid, coldVids);
```

Comment:
put extensive comment on this line, why this needs to be done,  give example witht that default object

Reply:
@kcudnik Done

Reply:
@kcudnik Done